### PR TITLE
Update demo request page

### PR DESCRIPTION
**Pull request by RonaldDijkstra**
This PR is for updating the ugly demo request pages and fix some other thingies 

Todos: 

@lennartdefacto: 
- [x] Check Dutch content and improve it

@me:
- [x] Check workings on different browsers
- [x] Add links from different pages to this one

@sn3p 
Review!

### CODE REVIEW

**Comment by RonaldDijkstra** [comment]
Deployed to https://staging.defacto.nl/demo-aanvragen/

**Comment by RonaldDijkstra** [comment]
@lennartdefacto it's all yours, content is only showing on 🇳🇱 website atm. Others can be done later, made an issue for that #1294

**Comment by RonaldDijkstra** [comment]
@sn3p R4R

**Comment by RonaldDijkstra** [comment]
Deployed again to https://staging.defacto.nl/demo-aanvragen/

Text still needs some scaling.

**Comment by RonaldDijkstra** [comment]
R4R

File: source/localizable/demo-aanvragen/_content.nl.md.erb [hunk 1]
```diff
@@ -0,0 +1,13 @@
+## Waaruit bestaat een Defacto demo?
+
+Een demo van Defacto staat niet op zichzelf. Onze experts kijken naar uw specifieke situatie en op welke manier wij samen met u kunnen bijdragen aan het ontwikkelen van uw medewerkers.
+
+Wij laten u tijdens een demo zien hoe u met behulp van onze producten:
+
+- eenvoudig een ultramoderne leeromgeving opzet en implementeert in uw organisatie, waarbij de gebruiker meer dan ooit centraal staat;
+- alle manieren van leren kunt ondersteunen binnen uw organisatie met CAPP LMS & aanverwante producten;
+- ervoor kunt zorgen dat al uw medewerkers inzicht hebben in waar ze voor het uitoefenen van hun werk bevoegd en bekwaam moeten zijn;
```
Comment (sn3p) [inline source/localizable/demo-aanvragen/_content.nl.md.erb]: ... bekwaam **voor** moeten zijn; ?

**Comment by lennartdefacto** [comment]
Added the word 'in' to the tekst to change it to 'bekwaam in moeten zijn'

**Pull request merged**